clang/include/clang/Basic/AttrDocs.td

```diff
@@ -1825,6 +1825,26 @@ be applied multiple times to the same loop.
   }];
 }
 
+def SYCLDeviceIndirectlyCallableDocs : Documentation {
+  let Category = DocCatFunction;
+  let Heading = "intel::device_indirectly_callable";
+  let Content = [{
+This attribute can only be applied to a functions and indicates that the
+function must be treated as device one and must be emitted even if it has no
+direct uses from other SYCL device functions. However, it cannot be applied to
+functions marked as 'static', functions declared within an anonymous
+space or class member functions.
```

Comment:
perhaps "anonymous namespace" is clearer but I might be incorrect

Reply:
Thanks, this indeed should be a "namespace" instead of "space"